pkg/exporter/api/config.go

```diff
@@ -0,0 +1,37 @@
+// +build minimal
+
+package api
+
+// We export below types in order for cli package to be able to read it from configuration file.
+type LogConfig struct {
+	Level  string
+	Output string
+}
+
+type MetricsConfig struct {
+	Path string
+}
+
+type ServerConfig struct {
+	Protocol string
+	Host     string
+	Port     string
+}
+
+type ExporterConfig struct {
+	Port    string
+	Log     *LogConfig
+	Metrics *MetricsConfig
+}
+
+type Config struct {
+	ZotServer   ServerConfig
+	ZotExporter ExporterConfig
+}
+
+func DefaultConfig() *Config {
+	return &Config{
+		ZotServer:   ServerConfig{Protocol: "http", Host: "localhost", Port: "8080"},
```

Comment:
What is this config for? If it is zot side, shouldn't you just read the zot server config?

Reply:
This is not related to zot binary in any way. We have a zot-exporter binary with separate CLI options & config.

Reply:
Use more generic names such as RegistryServer or Exporter?